samples/matter/light_bulb/README.rst

```diff
@@ -339,6 +339,61 @@ Upgrading the device firmware
 
 To upgrade the device firmware, complete the steps listed for the selected method in the :doc:`matter:nrfconnect_examples_software_update` tutorial in the Matter documentation.
 
+AWS IoT Integration
+===================
+
+The sample can be configured to communicate with AWS IoT Core to control attributes in supported clusters on the device.
+When properly setup, the sample will mirror these attributes in the AWS IoT shadow document.
+This makes it possible to remotely control the device using the `AWS IoT Device Shadow Service`_.
+The supported attributes are onoff and levelcontrol.
+
+To setup an AWS IoT instance and configure the sample, follow these instructions:
+
+1. Follow the AWS IoT library documentation to get the hostname, device ID, and certificates used in the connection: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/libraries/networking/aws_iot.html
+2. Set the CONFIG_AWS_IOT_BROKER_HOST_NAME and CONFIG_AWS_IOT_CLIENT_ID_STATIC options in :file:`overlay-aws-iot-integration.conf`
+3. Import the certificates under :file:`light_bulb/src/aws_iot_integration/certs`
+4. Build with :file:`overlay-aws-iot-integration.conf`:
+
+   .. code-block:: console
+
+      west build -p -b nrf7002dk_nrf5340_cpuapp -- -DOVERLAY_CONFIG="overlay-aws-iot-integration.conf"
+
+5. Flash the firmware, boot the sample and provision the device to wifi with the chip tool (or whatever works).
+6. Watch the device auto-connect to AWS IoT when IP is obtained and is able to maintain the connection.
+7. Use this bash function to populate the desired section of the shadow:
+
+   .. code-block:: console
+
+      function aws-update-desired() {
+            aws iot-data update-thing-shadow --cli-binary-format raw-in-base64-out --thing-name my-thing --payload "{\"state\":{\"desired\":{\"onoff\":$1,\"level_control\":$2}}}" "output.txt"
+      }
+
+   `aws-update-desired 0 0`, or `aws-update-desired 1 128` (onoff, levelcontrol)
+   Alternatively you can alter the device shadow directly via the `AWS IoT console`_.
+
+8. Watch the light bulb change state.
+   Note that local changes to the attributes will always take precedence over what is set in the the shadows desired state.
+
+.. note::
+   The integration layer has built in reconnection logic and will try to maintain the connection as long as the device is connected to the internet.
+   The reconnection interval can be configured by changing the CONFIG_AWS_IOT_RECONNECTION_INTERVAL_SECONDS option.
+
+Shadow Service interaction diagram
```

Comment:
Would it make sense to put these two diagrams before the test/setup steps to provide some introduction fro the customer before starting to use it? Then you could put the setup steps into separate section, like:

```
Testing Matter and AWS integration
----------------------------------
```